numba_cuda/numba/cuda/memory_management/nrt.py

```diff
@@ -113,10 +113,13 @@ def allocate(self, stream=None):
             self._compile_memsys_module()
 
         # Allocate space for NRT_MemSys
-        ptr, nbytes = self._memsys_module.get_global_symbol("memsys_size")
         memsys_size = ctypes.c_uint64()
+        ptr, nbytes = self._memsys_module.get_global_symbol("memsys_size")
+        device_memsys_size = ptr.device_ctypes_pointer
+        if USE_NV_BINDING:
+            device_memsys_size = device_memsys_size.value
```

Comment:
`device_memsys_size` would have previously been a `CUdeviceptr` object, but now will be an integer after this. We currently handle this in `cuda.bindings.driver` but I'm not sure if this is documented as behavior we guarantee. cc @leofang